frontend/src/components/interview/interview.tsx

```diff
@@ -0,0 +1,51 @@
+import { DataStatus, PaginationDefaultValue } from 'common/enums/enums';
+import { FC } from 'common/types/types';
+import { Spinner } from 'components/common/common';
+import {
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+  usePagination,
+  useParams,
+} from 'hooks/hooks';
+import { interviewActions } from 'store/actions';
+
+import { OtherApplications } from './components/components';
+import styles from './styles.module.scss';
+
+const Interview: FC = () => {
+  const { otherInterviews, totalOtherInterviewsNumber, dataStatus } =
+    useAppSelector((state) => state.interview);
+  const dispatch = useAppDispatch();
+  const { id } = useParams();
+  const { page, handlePageChange } = usePagination({
+    queryName: 'otherInterviews',
+  });
+
+  useEffect(() => {
+    dispatch(
+      interviewActions.getOtherByInterviewId({
+        interviewId: Number(id),
+        page,
+        count: PaginationDefaultValue.DEFAULT_COUNT,
+      }),
+    );
+  }, [page, id]);
+
+  if (dataStatus === DataStatus.PENDING) {
+    return <Spinner />;
+  }
+
+  return (
+    <div className={styles.wrapper}>
+      <OtherApplications
```

Comment:
```suggestion
      <OtherApplicationsTable
```
let's avoid an additional component